src/components/Layout/UserMenu.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+
+import Menu from "decentraland-gatsby/dist/components/User/UserMenu"
+import useFormatMessage from "decentraland-gatsby/dist/hooks/useFormatMessage"
+import { navigate } from "decentraland-gatsby/dist/plugins/intl"
+import MenuItem from "semantic-ui-react/dist/commonjs/collections/Menu/MenuItem"
+import Icon from "semantic-ui-react/dist/commonjs/elements/Icon/Icon"
+
+import locations from "../../modules/locations"
+
+const handleClickDocs = () => navigate(locations.docs())
+
+export default React.memo(function UserMenu() {
+  const l = useFormatMessage()
+
+  return (
+    <Menu
+      onClickSettings={undefined as any}
+      hasActivity={false}
+      onClickActivity={undefined as any}
+      onClickProfile={undefined as any}
```

Comment:
why is this required?

Reply:
Because `UserMenuProps` has this type, should we change that?

```
export type UserMenuProps = Pick<
  BaseUserMenuProps,
  | 'menuItems'
  | 'hasActivity'
  | 'onClickProfile'
  | 'onClickActivity'
  | 'onClickSettings'
> & {
  i18n?: Partial<UserMenuI18N>
  hideBalance?: boolean
}
```